client/component/sleeping-arrangements.jsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import styles from './sleeping-arrangements.css';
+
+const SleepingArrangements = (props) => (
+  <div className={styles.main}>
+    <div className={styles.title}>Sleeping arrangements</div>
+    <div className={styles.container}>
+    {props.info.sleep_arrangement ? props.info.sleep_arrangement.map( item => 
+      <div className={styles.box}>
+      <img src='https://s3-us-west-1.amazonaws.com/front-end-capstone/double-king-size-bed.png' className={styles.icon} />
+      <div className={styles.type}>{item === 'sofa bed' ? 'Common spaces' : `Bedroom ${props.info.bedroom_num}`}</div>
+      <div className={styles.bed}>{item}</div>
+      </div>
+    	) : null}
```

Comment:
This null looks awkward here. Is there a way to make this work without a ternary? Would it be possible to use an [Inline If with Logical && Operator](https://reactjs.org/docs/conditional-rendering.html#inline-if-with-logical--operator) instead?

Reply:
I don't understand what this `null` is for. It looks like it comes after the closing parenthesis for the map function?